packages/components/react-select-lib.d.ts

```diff
@@ -0,0 +1 @@
+export * from 'react-select';
```

Comment:
`d.ts` files should no be used for things like this, it should be used only for declaring type interfaces of files or modules (just import/export types). 

https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html

Reply:
Yes, thank you, forgot to change the extension after trying to do another thing. Tests were also failing.